modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -14,7 +14,10 @@ As a network administrator, you can configure a network attachment definition of
 
 .Procedure
 
-. Create a network attachment definition manifest.
+. Create a network attachment definition manifest. Ensure that you create the network attachment definition in the same namespace as the virtual machine.
+
+. Add the virtual machine to the network attachment definition.
```

Comment:
You could change this to "Add the virtual machine to the network attachment definition as in the following example:" and then get rid of the next sentence.

Reply:
Done.